test/validate/windows_validate.go

```diff
@@ -22,6 +23,17 @@ var windowsChecksMap = map[string][]check{
 	"cniv2": {
 		{"azure-vnet", azureVnetIps, privilegedLabelSelector, privilegedNamespace, azureVnetCmd},
 	},
+	"dualstack": {
+		{"azure-vnet", azureVnetIps, privilegedLabelSelector, privilegedNamespace, azureVnetCmd},
```

Comment:
We need to HNS network command here as well ?

Reply:
It's part of load-test here